annualreports/2025.md

```diff
@@ -77,21 +77,21 @@ The FPA has awarded in 2024 a total of USD 82 350.00 to developers, some through
 
 * **Bug triage maintainer**: The FPA now funds a bug triage maintainer, [Max](https://forum.freecad.org/memberlist.php?mode=viewprofile&u=56782), who is doing an incredible job in keeping the issues list active, clean, and making sure all submitted problems get the needed attention. There is now literally a "before Max" and "after Max" at FreeCAD.
 
 * **Release maintainer**: The FPA continued funding and extended the role of [Adrian](https://forum.freecad.org/memberlist.php?mode=viewprofile&u=19302) as a release manager. Adrian is now taking care of all the packages and installers on all platforms, making sure FreeCAD works everywhere, and is also taking care of bugfix releases (these minor releases that are done from time to time after main releases, to fix the most annoying bugs).
 
-* **Working on the forum and community guidelines**: As the FreeCAD community grows, community problems tend to increase. The FreeCAD forum (and other places where FreeCAD community members gather) sometimes suffer from members not behaving the friendly way we would like to encourage. This has made some potential new members avoid the forum because they do not see it as a safe or friendly enough place. Although this is a difficult question where there is no perfect solution, the FPA is working to develop possible solution strategies. Among other things, we have worked to improve the [community guidelines](https://www.freecad.org/codeofconduct.php), made it easier for people to find who to talk to in case of problems, and are trying to get more forum moderators to help alleviate the load on the shoulders of the few heroes who volunteer to do that job.
+* **Forum and community guidelines**: As the FreeCAD community grows, so too do community problems grow. The FreeCAD forum and other places where FreeCAD community members gather, sometimes suffer from members not behaving the respectful way we would all love to see always. This has made some potential new members avoid the forum because they do not see it as a safe or friendly enough place. Although this is a delicate question where there is no magical solution, we are working on, and are determined to keep working on, bettering the situation. Among other things, we have revised the [community guidelines](https://www.freecad.org/codeofconduct.php), made it easier for people to find who to talk to in case of problems, and are trying to get more forum moderators and help alleviate the load on the shoulders of the few heroes who volunteer to do that job.
```

Comment:
It looks to me like this is a reversion to the previous text (or at least a partial reversion) -- was that intentional, or an artifact of you and I editing at the same time? The same appears true for the next paragraph.

Reply:
Thought I was updating the latest.   Is current main correct?

Reply:
Yes - what caught my eye was the change back from "perfect" to "magical" ("magical" here felt to me almost facetious, so I changed it to "perfect" in my last revision). So I wanted to make sure the changes were intentional, and not some accidental git weirdness.